packages/ERTP/src/types.js

```diff
@@ -178,25 +179,12 @@
  */
 
 /**
- * @typedef {Object} Brand
- * The Brand indicates the kind of digital asset and is shared by
- * the mint, the issuer, and any purses and payments of this
- * particular kind. Fake digital assets and amount can use another
- * issuer's brand.
- *
- * @property {(allegedIssuer: any) => boolean} isMyIssuer Should be used with
- * `issuer.getBrand` to ensure an issuer and brand match.
- * @property {() => string} getAllegedName
- */
-
-/**
- * @typedef {Object} IssuerMaker
- * Makes Issuers.
+ * @callback MakeIssuerKit
+ * @param {string} allegedName
+ * @param {MathHelpersName} mathHelperName
```

Comment:
Should indicate that the `mathHelperName` is optional. (It defaults to `'nat'`)
```suggestion
 * @param {MathHelpersName=} mathHelperName
```